gcp-ts-gke/README.md

```diff
@@ -43,6 +42,7 @@ After cloning this repo, `cd` into it and run these commands. A GKE Kubernetes c
    ```bash
    $ pulumi config set nodeCount 5
    $ pulumi config set nodeMachineType n1-standard-2
+   $ pulumi config set masterVersion #any valid MasterVersion
```

Comment:
This is added for `gcp-ts-gke` but was removed from `gcp-ts-gke-hello-world`.

Based on the rest it seems like you'd want to remove this since the example pulls the `.latestMasterVersion` below

Reply:
I just moved it down into the optional config section in case they want to override.